src/NHibernate/Util/JoinedEnumerable.cs

```diff
@@ -70,10 +70,7 @@ public bool MoveNext()
 					}
 					else
 					{
-						// there are no items left to iterate over in the current
-						// enumerator so go ahead and dispose of it.
```

Comment:
Hi, @pawan-deepsource - autofix has remove the comment.

Reply:
Thanks for bringing it to my notice. Will track it internally and rollout a fix on priority.